packages/grafana-ui/src/types/datasource.ts

```diff
@@ -279,8 +279,10 @@ export interface QueryEditorProps<
   query: TQuery;
   onRunQuery: () => void;
   onChange: (value: TQuery) => void;
-  panelData: PanelData; // The current panel data
-  queryResponse?: PanelData; // data filtered to only this query.  Includes the error.
+  /*
+   * Contains query response filtered by refId and possible query error
+   */
+  queryResponse?: PanelData;
```

Comment:
don't know what the best name for this prop is. 

* data (like i PanelProps)
* queryResult
* panelData
* queryResponse

Reply:
data 👍

Reply:
The only counter to that is if `series` should really be called `data`... then we end up with data.data.   
```
export interface PanelData {
  state: LoadingState;
  series: DataFrame[];
  request?: DataQueryRequest;
  error?: DataQueryError;
}
```

Or we live with "series" as an akward naming artifact.  (I'm OK with this)

Reply:
well we can always rename series to frames. but maybe it's fine as series. Also ok with that.